pkg/aws/services/iam/oidc.go

```diff
@@ -337,3 +339,16 @@ func caThumbPrints(ep string) ([]string, error) {
 
 	return ret, nil
 }
+
+func removeDuplicates(tags []*iam.Tag) []*iam.Tag {
+	keys := make(map[string]bool)
```

Comment:
[super-minor]

In Go 1.21, there _may_ be a shorter version, roughly as follows.

```go
slices.SortFunc(names, func(a, b *iam.Tag) int {
	return cmp.Compare(a.Key, b.Key)
})
tags = slices.CompactFunc(tags, func(a, b *iam.Tag) bool {
	return a.Key == b.Key
})
```

No need to put this in, just wanted to mention it since I like the new, templated `slices` functionality.

Reply:
Thanks for the hint!